cdc/sink/mysql_params.go

```diff
@@ -266,6 +267,7 @@ func generateDSNByParams(
 	if params.timezone != "" {
 		dsnCfg.Params["time_zone"] = params.timezone
 	}
+	dsnCfg.Params["tidb_placement_mode"] = "ignore"
```

Comment:
Old versions of TiDB might not have `tidb_placement_mode`, so we need to consider a check like:

https://github.com/pingcap/tiflow/pull/4847/files#diff-7bc0811ccf84fd96ca586438530ad73f5cef7f3f4768821e50d9a2e4470bb942R275

Reply:
I think `checkTiDBVariable` can be used to check if the vairable `tidb_placement_mode` can be set